conanfile.py

```diff
@@ -254,6 +254,8 @@ def build(self):
             args.insert(0, "-shared")
         if self.settings.build_type == "Debug":
             args.append("-debug")
+        if self.settings.build_type == "Release":
```

Comment:
I think `elif`would be preferable.
Also, we should throw an error in the `else` line 266, because it now becomes an unknown build_type

Reply:
Shouldn't it build fine already because of the 266 line?

Reply:
it crashes on configurations which did not update settings.yml since [conan 1.3.0](https://docs.conan.io/en/latest/changelog.html?highlight=relwithdebinfo#april-2018):
```
  File "C:\Users\ContainerAdministrator\.conan\data\Qt\5.11.2\bincrafters\stable\export\conanfile.py", line 253, in build
    elif self.settings.build_type == "RelWithDebInfo":
  File "c:\tools\python3\lib\site-packages\conans\model\settings.py", line 94, in __eq__
    raise ConanException(bad_value_msg(self._name, other, self.values_range))
conans.errors.ConanException: Invalid setting 'RelWithDebInfo' is not a valid 'settings.build_type' value.
```